sdk/identity/identity/src/constants.ts

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * Current version of the `@azure/identity` package.
+ */
+export const SDK_VERSION = `azsdk-js-identity/2.1.0-beta.3`;
```

Comment:
I believe this should be just 2.1.0-beta.3

Also, didn't check the package.json, but friendly reminder to add it to the //constantPaths section so azsdk bot can update this